rootconf/default/etc/init.d/discover.sh

```diff
@@ -68,6 +69,7 @@ do_start() {
             install_remain_sticky_arch || {
                 echo "Error: problems making install boot mode sticky" > /dev/console
             }
+            sync;sync
```

Comment:
@cbrune 
For powerpc or arm,  it has stream buffer for MTD devices.  ONIE makes install boot mode sticky by writing variables to uboot-env.  So I think it should effect powerpc or arm, too.